public/js/components/Editor.js

```diff
@@ -101,12 +98,18 @@ const Editor = React.createClass({
     const closePanel = () => this.cbPanels[line].clear();
 
     if (isEnabled("conditionalBreakpoints") && bp && ev.metaKey) {
-      const { condition } = bp;
-      const panel = renderConditionalBreakpointPanel({
-        location, setBreakpointCondition, condition, closePanel
-      });
-
-      this.cbPanels[line] = this.editor.codeMirror.addLineWidget(line, panel);
+      if (!this.state.isCondBPOpen) {
```

Comment:
Tried this idea, it didn't work. Some weird codemirror thing is going on where the cbPanels[line] is still undefined by the time we re-render onGutterClick to close it. This might be directly related to the inability to focus, could be interesting to dive into codemirror in the future.